### ISSUE TITLE
Enabling navbar does not disable hardware buttons

### ISSUE THREAD

**Issue by eevleevs** [GravityBox/GravityBox#1974]
v.9.0.0-beta8 on A320/Lineage 16

**Comment by C3C0** [comment]
It's by design. Enabling software navigation bar doesn't have any relation to hardware button implementation.

**Comment by eevleevs** [comment]
Why would you want to keep both active? This way I get one soft button above another a hardware button with another command. I hit both half of the time, and the result is unpredictable.

**Comment by C3C0** [comment]
It's mainly because implementation can differ significantly between different device vendors so this feature would probably have to be targeted. As an example, on OnePlus devices, enabling software keys disables hardware keys automatically. Btw, doesn't Lineage already come with feature for enabling software keys instead of hardware ones?

**Comment by eevleevs** [comment]
Yes, but loading GB enables them again, even if navbar is disabled in GB.

**Comment by eevleevs** [comment]
For anyone interested, I managed to disable hardware keys again with https://play.google.com/store/apps/details?id=flar2.homebutton
